pkg/kubelet/kubelet.go

```diff
@@ -462,7 +463,19 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		}
 	}
 	httpClient := &http.Client{}
-	parsedNodeIP := net.ParseIP(nodeIP)
+
+	var nodeIPs []net.IP
+	for _, nodeIP := range strings.Split(nodeIP, ",") {
+		parsedNodeIP := net.ParseIP(strings.TrimSpace(nodeIP))
+		if parsedNodeIP == nil {
+			klog.Warningf("Could not parse --node-ip value %q; ignoring", nodeIP)
```

Comment:
Is it right to ignore this with no other signals?  I am not sure what is best but this seems likely to get lost in the noise

Reply:
Well, currently we ignore invalid `--node-ip` _without_ logging a warning, so this is at least an improvement...

Assuming we add `--node-ips` or `--node-ip-mode` in KEP-1664, we can make errors fatal there...